src/wallet/wallet.cpp

```diff
@@ -3138,6 +3143,27 @@ int64_t CWallet::GetOldestKeyPoolTime()
     return oldestKey;
 }
 
+void CWallet::MarkDestinationTransactionsDirty(const CTxDestination target_dst)
```

Comment:
Can do `const CTxDestination&` to avoid unnecessary copy, I think.

Reply:
done